sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -5,6 +5,7 @@ export enum Role {
   /**
    * AEST roles.
    */
+  AESTAdmin = "aest-admin",
```

Comment:
Is it the new role to allow the Ministry to trigger the manual reassessment?
If yes, it needs to be specific to the feature, for instance, `aest-manual-trigger-reassessment`.

Reply:
There's an AC: 'Set to keycloak role "admin"'. Did they mean group BusinessAdministrators?

Reply:
Yes @andrepestana-aot . Even thought it wasn't precise the AC meant to have a role to be part of business administrators group.

Reply:
Yes, they did.